frontend/app/routes/$lang+/_protected+/personal-information+/index.tsx

```diff
@@ -52,18 +55,22 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   const countryList = await getLookupService().getAllCountries();
   const regionList = await getLookupService().getAllRegions();
   const preferredLanguage = personalInformation.preferredLanguageId ? await getLookupService().getPreferredLanguage(personalInformation.preferredLanguageId) : undefined;
+  const maritalStatusList = await getLookupService().getAllMaritalStatuses();
+  const birthParsedFormat = personalInformation.birthDate ? toLocaleDateString(await parse(new Date(personalInformation.birthDate).toLocaleDateString(), 'dd/mm/yyyy', new Date()), await locale) : undefined;
```

Comment:
According to the [date-fns docs](https://date-fns.org/v3.6.0/docs/parse), `parse(..)` can return an invalid date:

> If parsing failed, Invalid Date will be returned.

So you have to check that the return value of `parse()` is valid. Have a look at `child-information.tsx` to see what I mean.

Reply:
Fixed: c643167653705749ea9866676ea2487dc7f57182

Reply:
Fixed: fa54d33af96bbc42837ec34ba79bb95bb345c315